playbooks/provisioner/openstack/plugins.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Setup BMC
+  sudo: yes
+  hosts: bmc
+  roles:
+      - { role: ovb/bmc, when: provisioner.nodes.bmc is defined }
```

Comment:
This is not a role, this is a playbook that calls the bmc role.
I will put the bmc role under provisioner/openstack/bmc.